### PR TITLE
fixing release version syntax

### PR DESCRIPTION
**Pull request by kalexand-rh**
Per @mazzystr in Slack.

@wking, does this sound right to you?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL?

**Comment by jianlinliu** [comment]
LGTM

File: modules/installation-mirror-repository.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ $ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
 $ export RELEASE_NAME="ocp-release" <6>
 ----
 <1> For `<release_version>`, specify the version number of {product-title} to
-install, such as `4.3.0`.
+install, such as `4.3.0-x86_64`.
```
Comment (wking) [inline modules/installation-mirror-repository.adoc]: Unless you're installing for another arch?

Comment (kalexand-rh) [reply modules/installation-mirror-repository.adoc]: @wking, I'm not aware that another arch is currently available for 4.3. Is the syntax for Z going to be `4.3.x-s390x`?

Comment (kalexand-rh) [reply modules/installation-mirror-repository.adoc]: @wking, who should I check with about suffixes for other arches?

Comment (wking) [reply modules/installation-mirror-repository.adoc]: Talk to ART about suffixes for other arches and official release timelines.  But personally I think we want #19266 so we don't need to care about what ART decides to use for tag names ;).

**Comment by kalexand-rh** [comment]
attaching to https://bugzilla.redhat.com/show_bug.cgi?id=1795544

@jianlinliu, I've made an update to match the bug. Please let me know if you want to merge this change or just wait for all the issues on #19266 to be resolved and merge that instead.

**Comment by jianlinliu** [comment]
I am okay to merget this change now.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL?

**Comment by bmcelvee** [comment]
LGTM

**Pull request merged**